esignet-core/src/main/java/io/mosip/esignet/core/dto/PushedAuthorizationRequest.java

```diff
@@ -64,4 +66,17 @@ public class PushedAuthorizationRequest implements Serializable {
 
     @RequestUri(strictNo = true)
     private String request_uri;
+
+    public void setClaims(String claimsJson) {
+        if (claimsJson == null || claimsJson.trim().isEmpty()) {
+            this.claims = null;
+            return;
+        }
+        try {
+            ObjectMapper mapper = new ObjectMapper();
+            this.claims = mapper.readValue(claimsJson, ClaimsV2.class);
+        } catch (IOException e) {
+            throw new IllegalArgumentException(ErrorConstants.INVALID_CLAIM, e);
+        }
+    }
```

Comment:
Not here it should be allowed in the claims_request_schema, can you once check oauthdetail request object.